pyproject.toml

```
@@ -0,0 +1,5 @@
+[tool.black]
+line-length = 99
+target-version = ['py37']
```

since this is what tests are currently doing anyways

```suggestion
target-version = ['py39']
```